Themes/default/Display.template.php

```diff
@@ -846,6 +846,10 @@ function template_single_post($message)
 
 			$base .= (isset($txt[$base . $count])) ? $count : 'n';
 
+			// Remove link if no cookies; session reference won't work
+			if (empty($_COOKIE))
+				$txt[$base] = strtr($txt[$base], array('<a href="%1$s">' => '', '</a>' => ''));
```

Comment:
```suggestion
				$txt[$base] = preg_replace('~</?a\b[^>]*>~', '', $txt[$base]);
```

This will ensure that the anchor tags are removed correctly even if the strings are changed in the future.

Reply:
Done